src/main/resources/TotemCode/SimpleTotemItemWebHomeSheet.xml

```diff
@@ -22,10 +22,184 @@
+        <separator> </separator>
+        <separators> ,|</separators>
+        <size>1</size>
+        <unmodifiable>0</unmodifiable>
+        <values>long|short|default|forbid</values>
+        <classType>com.xpn.xwiki.objects.classes.StaticListClass</classType>
+      </cache>
+      <code>
+        <disabled>0</disabled>
+        <name>code</name>
+        <number>2</number>
+        <prettyName>Code</prettyName>
+        <rows>20</rows>
+        <size>50</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.TextAreaClass</classType>
+      </code>
+      <name>
+        <disabled>0</disabled>
+        <name>name</name>
+        <number>1</number>
+        <prettyName>Name</prettyName>
+        <size>30</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
+      </name>
+      <parse>
+        <disabled>0</disabled>
+        <displayFormType>select</displayFormType>
+        <displayType>yesno</displayType>
+        <name>parse</name>
+        <number>4</number>
+        <prettyName>Parse content</prettyName>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.BooleanClass</classType>
+      </parse>
+      <use>
+        <cache>0</cache>
+        <disabled>0</disabled>
+        <displayType>select</displayType>
+        <multiSelect>0</multiSelect>
+        <name>use</name>
+        <number>3</number>
+        <prettyName>Use this extension</prettyName>
+        <relationalStorage>0</relationalStorage>
+        <separator> </separator>
+        <separators> ,|</separators>
+        <size>1</size>
+        <unmodifiable>0</unmodifiable>
+        <values>currentPage|onDemand|always</values>
+        <classType>com.xpn.xwiki.objects.classes.StaticListClass</classType>
+      </use>
+    </class>
+    <name>TotemCode.SimpleTotemItemWebHomeSheet</name>
+    <number>0</number>
+    <className>XWiki.JavaScriptExtension</className>
+    <guid>5eacd608-138e-4572-b890-250f57fa9fde</guid>
+    <property>
+      <cache>forbid</cache>
+    </property>
+    <property>
+      <code>require(['jquery'], function ($) {
+$(function(){
+//Initialize new item button
+$("input.createItemBtn").click(function(event){
+  if($.trim($("#createform-page").val()) != ""){
+   //Submit the form to create the item document
+   var saveURL = "$xwiki.getURL('TotemCode.EditContent','view','xpage=plain&amp;outputSyntax=plain&amp;action=saveItem')";
+   var data = $("form.xformInline").serialize();
+   saveURL = saveURL + "&amp;" + data;
+   $.ajax({
+      url:saveURL,
+      dataType: 'json',
+      success:function(rep, textStatus, jqXHR){
+         //$("#totemModal .modal-body").html(rep);
+         if(rep){ 
+             if(rep.results == false){
+                alert("The item name already exists!");
+             }
+             else
+             {
+                //Redirect
+                console.log(rep.results);
```

Comment:
The way I deal with console.log's is to add this at the top:
```javascript
var debug = function () { };
//var debug = function (x) { console.log(x); }; // uncomment to see debugging lines
```
then use `debug("blah blah message")`